thirdeye/thirdeye-frontend/app/pods/home/controller.js

```diff
@@ -11,6 +11,17 @@ export default Controller.extend({
     table: 'table-bordered table-condensed te-anomaly-table--no-margin'
   },
 
+  /**
+   * Default application for the application dropdown
+   * @type {Object} - the first application object from a list of applications
+   */
+  selectedApplication: computed(
+    'model.applications',
+    function() {
+      return this.get('model.applications')[0];
```

Comment:
I feel this name could be better named perhaps. Maybe something like `defaultApplication`, as it's just happens to be the selected as well. And i'm not sure we need to make this a computed if it's returning the first one anyway. Are we assuming here `model.applications` will update?

Reply:
As per our in-person conversation, I have changed this to set this variable in the route's `setupController` method.